cernopendata/templates/cernopendata_pages/front/main.html

```diff
@@ -0,0 +1,60 @@
+<div class="homepage-section main section align-items-center" data-anchor="main">
+
+    <div class="container container-main ">
+        <div class="main-inner">
+            <div class="cod-banner row">
+                <h2>
+                    <span>Explore more than <span><strong>1 petabyte</strong></span> <br>
+                    <span class="scnd-line">of open particle physics data !</span></span>
+                </h2>
+            </div>
+
+            <div class="cod-search row">
+                <form class="search-form container" action="{{config.THEME_SEARCH_ENDPOINT}}" role="search">
+                    <input type="text" class="form-control search" name="q" placeholder="{{_('Search')}}">
```

Comment:
Perhaps automatically focus on the Search input element, so that users can directly type?  They may be used to it from pages like Google...